### DISCUSSION TITLE
documentation for how to track total threads

### DISCUSSION THREAD

**Discussion by jjb** [sidekiq/sidekiq#6491] [General]
idea: add documentation to https://github.com/sidekiq/sidekiq/wiki/Commercial-FAQ for how to track total threads - it's interesting to see over time and then easier to keep track of if license is correct - here's example code with ruby-clock

```ruby
every '1 minute', name: 'Sidekiq Threads Monitor' do
  processes = Sidekiq::ProcessSet.new
  StatClient.value("Sidekiq: Threads Count", processes.total_concurrency)
end
```

**Comment by mperham** [comment]
Thanks for the example.
